src/dataServices/dynamoDbDataService.ts

```diff
@@ -35,7 +35,25 @@ import { DynamoDbBundleService } from './dynamoDbBundleService';
 import { DynamoDbUtil } from './dynamoDbUtil';
 import DynamoDbParamBuilder from './dynamoDbParamBuilder';
 import DynamoDbHelper from './dynamoDbHelper';
-import { getBulkExportResults } from '../bulkExport/bulkExportResults';
+import { getBulkExportResults, startJobExecution } from '../bulkExport/bulkExport';
+import { BulkExportJob } from '../bulkExport/types';
+
+const buildExportJob = (initiateExportRequest: InitiateExportRequest): BulkExportJob => {
+    const initialStatus: ExportJobStatus = 'in-progress';
+    return {
+        jobId: uuidv4(),
+        jobOwnerId: initiateExportRequest.requesterUserId,
+        exportType: initiateExportRequest.exportType,
+        groupId: initiateExportRequest.groupId ?? '',
+        outputFormat: initiateExportRequest.outputFormat ?? 'ndjson',
+        since: initiateExportRequest.since ?? '1800-01-01T00:00:00.000Z', // Default to a long time ago in the past

```

Comment:
I am a bit behind on this code flow in general but what does `since` represent? is it the `createdAt` time?

Reply:
It's a request query param that users can set. `since` is a filter that tells the export job to only pull records from `DDB` that have a `meta.lastUpdated` value that is newer than `since`